user_guide_src/source/general/controllers.rst

```diff
@@ -323,7 +323,7 @@ Validating $_POST data
 The controller also provides a convenience method to make validating $_POST data a little simpler, ``validate()`` that
 takes the current Request as the first instance, an array of rules to test against as the second parameter, and, optionally,
 an array of custom error messages to display if the items don't pass. The :doc:`Validation Library docs </libraries/validation>`
-has details on the format of the rules and messages arrays, as well as available rules.::
```

Comment:
This isn't actually a typo. The rst format requires double colons to do code-formatting on the following block of text.